packages/eupgrade/Makefile

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (C) 2020 Santiago Piccinini <spiccinini@altermundi.net>
+#
+# This is free software, licensed under the GNU General Public License v3.
+#
+
+include ../../libremesh.mk
+
+PKG_NAME:=eupgrade
+
+define Package/$(PKG_NAME)
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=$(PKG_NAME) provides semi automated firmware upgrades
+  MAINTAINER:=Santiago Piccinini <spiccinini@altermundi.net>
+  DEPENDS:=+lua +lime-system
```

Comment:
Is this yet another package which only on `lime.utils`?

Reply:
Yes!